amazon-sqs/5.8/modules/ROOT/pages/amazon-sqs-connector-example-large-message.adoc

```diff
@@ -0,0 +1,427 @@
+. In the *Send message* configuration, click *+* next to the *Connector configuration* field to add a global element.
+. Configure the global element for *Send message*:
++
+[%header%autowidth.spread]
+|===
+|Field |Value
+|Name | Name used to reference the configuration. This example uses `Amazon_SQS_Large_Payload_Configuration`.
+|Session Token | Session token used to validate the temporary security credentials
+|Access Key |Alphanumeric text string that uniquely identifies the user who owns the account
+|Secret Key |Key that acts as a password
+|Region Endpoint | Queue region
+|Default Global Queue URL | Default Amazon SQS queue URL
+ credentials
+|===
++
+For example:
++
+image::amazon-sqs-studio-global-config-new.png[send-global-config]
++
+. In the *Connection* section, click on the *Advanced* tab.
+
+. In the *Large Payload Support* field, choose either:
+
+* *Expression or Bean Reference* 
+* *Edit Inline* 
+.. Configure the parameters as follows:
++
+[%header%autowidth.spread]
+|===
+|Field |Description
+|Bucket | Name of the AWS S3 bucket to use for storing large message payloads. The bucket must already be created and configured in AWS S3. Enabling this feature incurs additional charges for using AWS S3.
+|Message Size Threshold | The message size threshold value for storing message payloads in the AWS S3 bucket. The default value for message size threshold is 256 KB and the maximum threshold size value is 256KB. The maximum message size is 2GB.
+|Message Size Threshold Unit | Data unit for the message size threshold
+|===
++
+For example:
++
+image::amazon-sqs-studio-large-payload-support.png[Configure Large Payload Support]
++
+. At the base of the Studio canvas, click *Configuration XML* to view the corresponding XML:
++
+[source,dataweave,linenums]
+----
+<sqs:config name="Amazon_SQS_Configuration"
+  doc:name="Amazon SQS Configuration"
+	defaultQueueUrl="${sqs.queueUrl}">
+	<sqs:basic-connection accessKey="${sqs.accessKey}"
+	secretKey="${sqs.secretKey}"
+	region="us-east-1"
+	sessionToken="${sqs.sessionToken}">
+	    <sqs:large-payload-support bucket="large-sqs-payload-bucket"
+            messageSizeThreshold="256" messageSizeThresholdUnit="KB" />
+    </sqs:basic-connection>
+</sqs:config>
+----
++
+. Configure the following fields in the properties window:
++
+[%header%autowidth.spread]
+|===
+|Field |Value
+|Display Name |Name for the connector operation
+|Connector Configuration |Global configuration you previously created for the *Send message* operation
+|Message |`payload`
+|Queue url |Amazon SQS queue URL.
+
+If provided, the value of this field takes precedence over the value of the *Default Global Queue URL* field on the *Global Configuration Elements* window.
+|===
++
+For example:
++
+image::amazon-sqs-send-message.png[send-message]
+
+=== Add and Configure a Logger Component
+
+Add and configure a *Logger* component to display the response in the Mule console:
```

Comment:
The response to...? Adding and Configuring the SQS Send Message Operation, the parent task?